sqlserver/tox.ini

```diff
@@ -2,14 +2,14 @@
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}-linux-{2017,2019}-{single,ha}
+    py{27,38}-linux-{2017,2019}-{single,ha,high-cardinality}
     # test the full combination of python-version/driver against a the latest sql server version
     # ideally we'd test this against all sql server versions but that makes the test take too long and time out.
     # time out. until we're able to modify and parallelize the work we'll limit the per-driver tests to only a single
     # sqlserver version
     py{27,38}-windows-{SQLOLEDB,SQLNCLI11,MSOLEDBSQL,odbc}-2019-single
     # older sql server versions tested on only a single python version and driver
-    py38-windows-odbc-{2016,2017}-single
+    py38-windows-odbc-{2016,2017}-{single,high-cardinality-windows}
```

Comment:
Can we update to have the high cardinality test only on a single set of envs in order to avoid significantly extending the build time? It's at 84 minutes now so we're nearing the recently raised limit. 

Instead of putting in
```
py{27,38}-linux-{2017,2019}-{single,ha,high-cardinality}
py38-windows-odbc-{2016,2017}-{single,high-cardinality-windows}
```

Can we add a new line so the high-cardinality is done only once per OS? 
```
py38-{linux,windows}-odbc-2019-high-cardinality
```